.github/workflows/reusable_tutorials.yml

```diff
@@ -0,0 +1,64 @@
+name: Reusable Tutorials Workflow
+
+on:
+  workflow_call:
+    inputs:
+      smoke_test:
+        required: false
+        type: boolean
+        default: true
+      use_stable_pytorch_gpytorch:
+        required: false
+        type: boolean
+        default: false
+      use_stable_ax:
+        required: false
+        type: boolean
+        default: false
+
+jobs:
+  tutorials:
+    name: Run tutorials
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Fetch all history for all tags and branches
+      # We need to do this so setuptools_scm knows how to set the BoTorch version.
+      run: git fetch --prune --unshallow
+    - if: ${{ !inputs.use_stable_pytorch_botorch }}
+      name: Install latest PyTorch & GPyTorch
+      run: |
+        pip install git+https://github.com/cornellius-gp/gpytorch.git
```

Comment:
We should probably install the pytorch nightly here? Otherwise gpytorch will resolve the requirement upon install and then install the latest compatible release.